lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -532,32 +548,31 @@ func processAnonymousVolumes(volumes map[string]struct{}) ([]volumeFields, error
 
 // processSpecifiedVolumes parses fields for volumes specified explicitly during a
 // create/run (such as -v volume-name:/dir) and creates them if they don't already exist.
-func processSpecifiedVolumes(volumes []string) ([]volumeFields, error) {
-	var volumeFields []volumeFields
+func processSpecifiedVolumes(volumes []string) ([]volumeFields, error, error) {
```

Comment:
Just repeating the above;  lets try to avoid returning more than one `error`.  If you need to find the condition the error is being thrown from, then it sounds like you need a specific error type to assert.